### ISSUE TITLE
WebView.evaluate broken on macOS 10.14

### ISSUE THREAD

**Issue by poke1024** [pybee/toga#600]
## Expected Behavior
On macOS 10.14, `WebView.evaluate` should work.

## Current Behavior
Calling `WebView.evaluate()` brings up this error:

```
  File "/path/to/python3.7/site-packages/rubicon/objc/api.py", line 591, in __getattr__
    type(self).__module__, type(self).__qualname__, self.objc_class.name, name)
AttributeError: rubicon.objc.api.ObjCInstance TogaWebView has no attribute stringByEvaluatingJavaScriptFromString
```

Note: it seems that the underlying `stringByEvaluatingJavaScriptFromString` is deprecated (https://developer.apple.com/documentation/webkit/webview/1408429-stringbyevaluatingjavascriptfrom?language=objc) and will vanish completely in macOS 10.15. From there on, it seems, only asynchronous evaluation of javascript will be possible via `evaluateJavaScript`.

## Steps to reproduce
See accompanying `reproduce.py`.

Also see accompanying `async.py` for a workaround/new implementation of `WebView.evaluate` using `evaluateJavaScript` and `asyncio`.

[source.zip](https://github.com/pybee/toga/files/2442846/source.zip)

## Your Environment

* Python Version (list the specific version number) 3.7.0

* Operating System and Version (select from the following and list the specific version number; if your OS is not listed, list that as well)

  - [x] macOS - version: 10.14

  - [ ] Linux - distro:   - version:
  - [ ] Windows - version:
  - [ ] Other - name:  - version:

* Toga Target (the type of app you are trying to generate)
 
  - [ ] android
  - [x] cocoa
  - [ ] django 
  - [ ] gtk
  - [ ] iOS
  - [ ] tvOS
  - [ ] watchOS
  - [ ] winforms 
  - [ ] win32
  - [ ] Other (please specify)

**Comment by freakboy3742** [comment]
Thanks for the report. It appears I was a bit hasty making this change to WKWebView; there are a bunch of other changes in WKWebView around visibility of file:// URLs that also cause a regression. 

Looks like we'll have to revert to a WebView based implementation for at least the short term.

**Comment by freakboy3742** [comment]
This should be resolved by #605.
